__tests__/server.test.js

```diff
@@ -13,11 +13,10 @@ beforeEach(() => {
 
 afterAll(() => connection.end());
 
-
 describe('GET', () => {
     test('404 - response with error message when given a url that doesnt exist on server', () => {
         return request(app)
-        .get('/api/pokeomon-news')
+        .get('/pokeomon-news')
```

Comment:
Technically doesn't matter but I quite like having /api/nonsense as it shows it's trying to be used correctly, doesn't really matter though :D